nss-day-cohort-41/nutshell-asking-for-roses
# Implement design on login page

# Description
Implemented design on the login page to create a clean and friendly user experience. This item addresses ticket #1.

Fixes # (issue)
This update also fixes a bug where, when a user signs up and mismatches the password, they receive an error, but their info still POSTs.

## Type of change
Please delete options that are not relevant.
# Testing Instructions
Serve the page. Changes will be immediately visible.
Please test all login and signup functions as well to make sure functionality was maintained.

# Checklist:
